src/Microsoft.AspNet.Mvc.Core/ActionResults/FilePathResult.cs

```diff
@@ -208,22 +212,50 @@ private IFileProvider GetFileProvider(IServiceProvider requestServices)
             return FileProvider;
         }
 
-        private static async Task CopyStreamToResponse(
-            string fileName,
+        private Task CopyPhysicalFileToResponseAsync(
+            HttpResponse response,
+            string physicalFilePath,
+            CancellationToken cancellationToken)
+        {
+            var sendFile = response.HttpContext.GetFeature<IHttpSendFileFeature>();
+            if (sendFile != null)
+            {
+                return sendFile.SendFileAsync(
+                    physicalFilePath,
+                    offset: 0,
+                    length: null,
+                    cancellation: cancellationToken);
+            }
+            else
+            {
+                var fileStream = new FileStream(
```

Comment:
I believe this is because it was following the below code:
https://github.com/aspnet/FileSystem/blob/0d2d9b1913175a764faaa57dc40de3e3761797c9/src/Microsoft.AspNet.FileProviders/PhysicalFileProvider.cs#L292